Components/Metrics/Impact/itkBSplineInterpolateVectorImageFunction.hxx

```diff
@@ -48,9 +48,9 @@ BSplineInterpolateVectorImageFunction<TImage, TInterpolator>::Evaluate(typename
                                                                        std::vector<unsigned int> subsetOfFeatures) const
 {
   std::vector<float> result;
-  for (int i = 0; i < subsetOfFeatures.size(); ++i)
+  for (const unsigned int feature : subsetOfFeatures)
```

Comment:
OK? Or should the loop variable be named something like `featureNumber`? As in:

    for (const unsigned int featureNumber : subsetOfFeatures)